CMakeLists.txt

```diff
@@ -411,7 +411,7 @@ if(${VELOX_ENABLE_DUCKDB})
 endif()
 
 set_source(fmt)
-resolve_dependency(fmt 9.0.0)
```

Comment:
nit: I don't think we need to specify the version any more.

Reply:
It's a minimum version so we should keep it. The issue is that fmt doesn't provide a proper version config that would prevent us from picking up an incompatible version.